tests/foreman/installer/test_installer.py

```diff
@@ -1350,12 +1350,17 @@ def common_sat_install_assertions(satellite):
 def install_satellite(satellite, installer_args):
     # Register for RHEL8 repos, get Ohsnap repofile, and enable and download satellite
     satellite.register_to_cdn()
-    satellite.download_repofile(
-        product='satellite',
-        release=settings.server.version.release,
-        snap=settings.server.version.snap,
-    )
-    satellite.execute('dnf -y module enable satellite:el8 && dnf -y install satellite')
```

Comment:
why removing this module enable step? I think it would be required for install_satellite

Reply:
That was by mistake